build.rs

```diff
@@ -54,6 +54,27 @@ pub fn main() {
 
     for krate in crates {
         match krate.as_ref() {
+            "uu_test" => {
```

Comment:
please add a comment explaining the why :)

Reply:
Added